src/Microsoft.AspNet.Identity.EntityFramework/IdentityDbContext.cs

```diff
@@ -48,62 +48,59 @@ protected IdentityDbContext()
         protected override void OnModelCreating(ModelBuilder builder)
         {
             builder.Entity<TUser>(b =>
-                {
-                    b.HasKey(u => u.Id);
-                    b.HasIndex(u => u.NormalizedUserName).HasName("UserNameIndex");
-                    b.HasIndex(u => u.NormalizedEmail).HasName("EmailIndex");
-                    b.ToTable("AspNetUsers");
-                    b.Property(u => u.ConcurrencyStamp).IsConcurrencyToken();
-
-                    b.Property(u => u.UserName).HasMaxLength(256);
-                    b.Property(u => u.NormalizedUserName).HasMaxLength(256);
-                    b.Property(u => u.Email).HasMaxLength(256);
-                    b.Property(u => u.NormalizedEmail).HasMaxLength(256);
-                    b.HasMany(u => u.Claims).WithOne().HasForeignKey(uc => uc.UserId);
-                    b.HasMany(u => u.Logins).WithOne().HasForeignKey(ul => ul.UserId);
-                    b.HasMany(u => u.Roles).WithOne().HasForeignKey(ur => ur.UserId);
-                });
+            {
+                b.HasKey(u => u.Id);
+                b.HasIndex(u => u.NormalizedUserName).HasName("UserNameIndex");
+                b.HasIndex(u => u.NormalizedEmail).HasName("EmailIndex");
+                b.ToTable("AspNetUsers");
+                b.Property(u => u.ConcurrencyStamp).IsConcurrencyToken();
+
+                b.Property(u => u.UserName).HasMaxLength(256);
+                b.Property(u => u.NormalizedUserName).HasMaxLength(256);
+                b.Property(u => u.Email).HasMaxLength(256);
+                b.Property(u => u.NormalizedEmail).HasMaxLength(256);
+                b.HasMany(u => u.Claims).WithOne().HasForeignKey(uc => uc.UserId).OnDelete(DeleteBehavior.Cascade);
```

Comment:
@HaoK Does User -> Roles need to have IsRequired? It looks like it should be required by default.